cf_remote/commands.py

```diff
@@ -476,10 +476,25 @@ def uninstall(hosts):
     return errors
 
 def deploy(hubs, directory):
-    assert(directory.endswith("/masterfiles"))
-    assert(os.path.isfile(directory + "/autogen.sh"))
+    directory = os.path.abspath(os.path.expanduser(directory))
+    log.debug(f"Deploy directory expanded to: {directory}")
+    if directory.endswith("/"):
+        directory = directory[0:-1]
```

Comment:
`directory.rstrip("/")` could replace the two lines above

Reply:
Not technically the same (rstrip would remove multiple slashes), but I don't think it matters too much here.

Reply:
https://github.com/cfengine/cf-remote/pull/8